experiments/pages/docusign/autoPopulationTutorial.md

```diff
@@ -0,0 +1,135 @@
+ 
+
+##### Skuid Configure: 
+1.	Go to Data Sources > Authentication Providers
+2.	Create a new Authentication Provider with a unique name and DocuSign Provider type. 
+3.	Set the Authorize Endpoint URL and Token Endpoint URL to:
+a.	 Authorize: https://account-d.docusign.com/oauth/auth
+b.	Token: https://account-d.docusign.com/oauth/token
+4.	Set the Client Id to the Integrator Key Id defined in DocuSign
+5.	Set the client secret to the Secret Key you saved from DocuSign earlier. Save! 
+6.	Now go to Data Sources > Data Sources
+7.	Create a New Data Source with a unique name and with the REST Data Source Type
+8.	Set the URL to  https://demo.docusign.net/restapi and the Authentication Provider to DocuSign. Save. 
+
+
+##### Skuid Page:
+1.	Create a REST api model on the DocuSign data source with Read/Write Model Behavior.
+2.	On this new model, select Methods>insert
+3.	Set values as follows: 
+a.	type =Insert
+b.	url = /accounts/{{AccountId}}/envelopes/
+c.	HTTP verb =POST 
+d.	Send new field values = “As templated request body” 
+4.	On this model, create a field for every piece of data that you want to insert into the DocuSign template. Also include fields for undefined recipients (name and email). 
+5.	For each field you create, set the default value to point to the data you want to merge (Field from another model is the easiest way to do this).
+6.	Get ready to create your custom request body!
+
+
+##### Creating Your Custom Request Body:
+Because DocuSign’s REST API is fairly complex, creating a custom request body with merge variables is the way to go. (View an example below!)
+You can basically follow the format given in the DocuSign API Explorer:  https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=Envelopes&operations=create but you will only need the following fields: 
+1.	Envelope
+a.	Status  - either sent or created (for a draft)
+b.	Template Id  - referenced above. 
+c.	Email Subject - define the subject of the email to be sent with the docusign doc. 
+2.	Recipient
+a.	name
+b.	Email
+c.	roleName - same as defined in the DocuSign template
+d.	recipientId - the best way to get this is to do a get request on the api explorer  (https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=EnvelopeRecipients&operations=list) 
+e.	routingOrder (optional) - can also be defined in DocuSign, but it can be helpful to conditionally choose the routing order in Skuid. 
+3.	Recipient tab
+a.	tabLabel - Data Label defined in DocuSign
+b.	value - the value you want to insert (use merge syntax here!)
+c.	font (optional) - choose a font to stylize your text. 
+i.	Supported Fonts: Arial, Arial, ArialNarrow, Calibri, CourierNew, Garamond, Georgia, Helvetica,   LucidaConsole, Tahoma, TimesNewRoman, Trebuchet, Verdana, MSGothic, MSMincho, Default.
+
+##### See it in action!: 
+1.	In your Skuid page:
+2.	Make sure there is a new row on the “POST” model. 
```

Comment:
Instead of saying "Make sure..." here, maybe we could recommend that you remove any existing rows in the Model, and create a new row?